MANIFEST.in

```diff
@@ -1,23 +1,17 @@
-graft docs
 graft src
-graft tests
+exclude src/mlx/warnings/__version__.py
```

Comment:
This file should not be excluded as it enables setuptools to work without git and stuff, so it should be packaged together.

Reply:
Huh, I don't understand. `setuptools_scm` generates this file and it ends up in the package. Without git and stuff, the version number cannot be determined during installation, but that's not what you mean, is it?

If I change this to `include`, the following command fails:

```
lists of files in version control and sdist do not match!
missing from VCS:
  src/mlx/warnings/__version__.py
  ```